pages/common/g++.md

```diff
@@ -0,0 +1,19 @@
+# G Plus Plus
+
+> G++ is a compiler for C++. Member of GNU GCC.
+
+- Compile a source code file into an executable binary:
+
+`g++ {{input_source.cpp}} -o {{output_executable}}`
+
+- Activate output of (almost) all errors and warnings:
```

Comment:
Why "almost" ?

Probably `Display all warnings and errors in the output:` sounds better.

Reply:
`-Wextra` enables more warnings, and even with both it still doesn't cover [these warnings](https://gcc.gnu.org/onlinedocs/gcc-4.4.0/gcc/Warning-Options.html).

Reply:
Ok, we can keep the "almost" then.